examples/server/README.md

```
@@ -4,6 +4,48 @@ You found the example backend implementation! 🎉
+cd examples/server
+dotnet run
+# Listening on http://localhost:5200
+```
+
+**2. Start the updater in Debug** (Visual Studio: open `vīcĭus.sln`, set the
+`vīcĭus` project as the startup project, select **Debug | x64**, press F5).
+
+The project ships a committed `src/vīcĭus.vcxproj.user` that passes these
+arguments automatically:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)